t8n/src/utils.rs

```diff
@@ -82,6 +81,8 @@ pub fn handle_transactions(starknet: &mut Starknet, transactions: Vec<Broadcaste
             }
         }
     }
+    let state_diff = starknet.state.commit_with_diff().unwrap();
+    let _ = starknet.generate_new_block(state_diff.clone());
```

Comment:
Should we modify this `generate_new_block` fn to not retun anything?
 If not then I guess we should handle it by `match` block